.github/workflows/update-pots.yml

```
@@ -0,0 +1,21 @@
+on:
+  schedule:
+    - cron: '*/30 * * * *'
+  push:
+    branches:
+      - '*'
+jobs:
+  update-pots:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@master
+      - uses: actions/setup-python@master
+        with:
+          python-version: 3
+      - run: git config --local user.email github-actions@github.com
+      - run: git config --local user.name "GitHub workflow's update-pots job"
```

Not mandatory, but that's what I see in the webs and that's what I use in my repos.

```suggestion
      - run: git config --local user.name github-actions[bot]
```